connectivity/builder/builder.go

```diff
@@ -79,23 +79,86 @@ type testBuilder interface {
 	build(ct *check.ConnectivityTest, templates map[string]string)
 }
 
-// NetworkPerformanceTests builds the network performance connectivity tests.
-func NetworkPerformanceTests(ct *check.ConnectivityTest) error {
-	return getTests(ct, []testBuilder{networkPerf{}})
+// GetTestSuites returns a slice of connectivity test builder functions.
+// Each function contains set of tests according to the provided parameters.
```

Comment:
This godoc doesn't really describe what this function is doing. 

This function returns a slice of functions, which when those functions are invoked, append `testBuilder` structs to a slice which are each invoked by an IoC pattern.

The key distinction here _is that this slice of functions are not the test builders_. They are two levels removed from the invocation of the test builders.

A better docstring would be:

```
// GetTestSuites returns a slice of functions, that when invoked, result in a
// collection of tests being built. These functions use helper methods to add various
// collections of test builders depending upon the provided parameters.
```

This function is in the `builder` package, and it's return value is called `builder` below, but we already have a new type named `testBuilder`, which is a struct that represents a test which can be built.

The type of these functions would therefore be better named `suiteBuilder` or similar.

Reply:
Nice, thank you very much!
Fixed.